src/components/projectDetail/DonateDock.tsx

```
@@ -533,292 +635,328 @@ const DonateDock = () => {
+                      {/* 왼쪽 영역: 상품 카드 목록 (1/3 너비) - 스크롤 가능하도록 수정 */}
+                      <div className="md:col-span-1">
+                        <div className="h-[500px] overflow-y-auto pr-2 pt-4 space-y-6">
+                          {projectOptions.map((option) => {
+                            const isSelected = selectedOptions.includes(option.id)
+                            return (
+                              <div
+                                key={option.id}
+                                className={`relative flex cursor-pointer items-center rounded-xl border p-4 transition-all ${
+                                  isSelected
+                                    ? "border-2 border-main-color"
+                                    : "border-gray-border hover:border-secondary-color-dark"
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)